app/mishki-index.html

```diff
@@ -0,0 +1,146 @@
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/style.min.css">
+  <title>Mishka</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="nav--wrapper">
+        <img class="nav__logo logo" src="/app/imajes/jpg/mishka-logo-mobile.svg copy.png" alt="Мишка">
+        <button class="main-nav__togle" type="button"><?xml version="1.0" encoding="iso-8859-1"?>
+          <svg class="svg__icon" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+             width="35px" height="30px" viewBox="0 0 612 612" style="enable-background:new 0 255 255 255;" xml:space="preserve">
+            <g>
+              <path d="M604.267,194.727c4.257,0,7.733-3.644,7.733-7.733v-40.169c0-4.256-3.283-7.733-7.733-7.733H7.733
+                c-4.256,0-7.733,3.644-7.733,7.733v40.169c0,4.256,3.283,7.733,7.733,7.733H604.267z"/>
+              <path d="M0,326.084c0,4.256,3.283,7.733,7.733,7.733h596.533c4.256,0,7.733-3.645,7.733-7.733v-40.169
+                c0-4.284-3.283-7.733-7.733-7.733H7.733c-4.256,0-7.733,3.645-7.733,7.733V326.084z"/>
+              <path d="M0,465.175c0,4.256,3.283,7.733,7.733,7.733h596.533c4.256,0,7.733-3.645,7.733-7.733v-40.169
+                c0-4.256-3.283-7.732-7.733-7.732H7.733c-4.256,0-7.733,3.644-7.733,7.732V465.175z"/>
+            </g>
+          </svg>
+          </button>
+        </div>
+        <ul class="nav__list site-list">
+          <li class="site-list__item"><a href="#">Каталог товаров</a></li>
+          <li class="site-list__item"><a href="#">Вязание на заказ</a></li>
+          <li class="site-list__item"><img class="nav__list-logo logo" src="/app/imajes/jpg/mishka-logo-desktop.svg.png" alt="Мишка"></li>
+          <li class="site-list__item"><label>
+            <span class="search__logo"><svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+              width="15px" height="15px" viewBox="0 0 612.08 612.08" style="enable-background:new 0 0 612.08 612.08;"
+              xml:space="preserve">
+           <g>
+             <path d="M237.927,0C106.555,0,0.035,106.52,0.035,237.893c0,131.373,106.52,237.893,237.893,237.893
+               c50.518,0,97.368-15.757,135.879-42.597l0.028-0.028l176.432,176.433c3.274,3.274,8.48,3.358,11.839,0l47.551-47.551
+               c3.274-3.274,3.106-8.703-0.028-11.838L433.223,373.8c26.84-38.539,42.597-85.39,42.597-135.907C475.82,106.52,369.3,0,237.927,0z
+                M237.927,419.811c-100.475,0-181.918-81.443-181.918-181.918S137.453,55.975,237.927,55.975s181.918,81.443,181.918,181.918
+               S338.402,419.811,237.927,419.811z"/>
+           </g>
+           </svg>
+           </span>Поиск по сайту
+            <input class="search" type="search" placeholder="Поиск по сайту">
```

Comment:
```suggestion
            <input 
                class="search" 
                type="search" 
                placeholder="Поиск по сайту"
            >
```

когда 3 и больше аттрибута, тогда переносим (либо если большие 2 аттрибута)
ну намного легче читать как набор параметров, а не как спагетти которая горизонтально убегает